content/project/approved-trademark-usage.adoc

```diff
@@ -0,0 +1,128 @@
+---
+layout: simplepage
+title: "Approved trademark usage"
+section: project
+---
+
+This document lists the use of "Jenkins" mark approved by the link:https://wiki.jenkins.io/display/JENKINS/Governance+Meeting+Agenda[project meeting], along with its meeting notes.
```

Comment:
Note that it lifts the page protection. The former page is editable only by Board members while the new one can be edited my anyone with Write permissions to the repository